joint_angles:
    vertical_pose: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    #intermediate_pick_pose: [179.0, 17.0, 182.0, 231.0, 0.0, 56.0, 92.0]
    look_at_ground_pose: [290.463104248, 352.912414551, 347.893951416, 246.30140686, 1.53782832623, 299.26449585, 98.9884338379]
    intermediate_place_pose: [239.447753906, 345.333770752, 346.232299805, 255.687530518, 3.58446717262, 297.186737061, 89.6425933838]
    place_pose: [236.549240112, 332.791503906, 348.780487061, 244.14201355, 7.72167634964, 321.007019043, 43.3857421875] 
